### DISCUSSION TITLE
is it possible to use LlamaTale with openblas?

### DISCUSSION THREAD

**Discussion by kroryan** [neph1/LlamaTale#34] [General]
i would like to know if is possible to use llamatale with openblas and how to do it

**Comment by neph1** [comment]
LlamaTale runs independently of your LLM backend. So the answer is "yes", but how you do it depends on how your LLM is served. How are you running your LLM today? I mean, is it koboldcpp, oobabooga or something else?

**Reply by kroryan** [reply]
well im using koboldcpp, so i think i will have to just use a koboldcpp make with openblas right?
